driver/csiplugin/controllerserver.go

```diff
@@ -1907,7 +2040,7 @@ func (cs *ScaleControllerServer) DeleteVolume(ctx context.Context, req *csi.Dele
 	}
 
 	relPath := ""
-	if volumeIdMembers.StorageClassType == STORAGECLASS_ADVANCED || volumeIdMembers.VolType == FILE_SHALLOWCOPY_VOLUME {
+	if volumeIdMembers.StorageClassType != STORAGECLASS_CLASSIC || volumeIdMembers.VolType == FILE_SHALLOWCOPY_VOLUME {
```

Comment:
are we not deleting the coskeys created in the create flow in delete flow ?

Reply:
Agree, ideally we should delete the keys for cache volume under deletion flow, will fix that under scale-core 7105